curation_service/app.py

```diff
@@ -286,7 +264,11 @@ def update_curations():
                  ('pa_hash', 'stmt_hash'), 'source_hash']
 
     # Build up the curation dict.
+    db_key = "primary"
     db = get_db('primary')
+    if db is None:
+        raise RuntimeError(f"unable to get database: {db_key}")
```

Comment:
this will raise if your configuration for the primary database is old  (like mine, ATM)